index-de.html

```diff
@@ -76,7 +76,7 @@ <h2>Die längere Fassung</h2>
       Solltest du dich belästigt fühlen oder mitbekommen, dass jemand anders
       belästigt wird oder sonstige Probleme auftreten, wende dich bitte sofort
       an ein Mitglied des Veranstaltungspersonals. Diese sind an ihren
-      entsprechend markierten T-Shirts schnell zu erkennen.
+      entsprechend markierten T-Shirts und/oder Ausweisen schnell zu erkennen.
```

Comment:
Is this still gender neutrality only? What is the aim of this change? 🙂

Reply:
It aims to make the sentence more clear, but agreed, not necessarily related to this PR. There are other rephrasing changes which make this more applicable to native German.